CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 - Add an `AccommodationOption` model (#1731, #1736)
 
 ### Changed
+- Converted icons from gif/png to SVG (#1522)
```

Comment:
This should be the ID of the PR, not of the ticket. (This allows people to click on the ID in the changelog and directly see the changes.)

Also, please move this to the `Added` section.